.eslintrc

```diff
@@ -1,31 +1,71 @@
 {
   "root": true,
-  "parser": "babel-eslint",
-  "extends": ["airbnb-base", "prettier", "plugin:import/errors", "plugin:import/warnings"],
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "prettier"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 2020,
+    "sourceType": "module"
+  },
+  "plugins": ["@typescript-eslint", "prettier"],
+  "env": {
+    "browser": true,
+    "node": true,
+    "es6": true
+  },
   "rules": {
     "prettier/prettier": "error",
-    "import/prefer-default-export": "off",
-    "import/no-extraneous-dependencies": [
+    "@typescript-eslint/naming-convention": [
       "error",
       {
-        "devDependencies": true,
-        "optionalDependencies": true
+        "format": ["PascalCase"],
+        "selector": "typeLike",
+        "custom": {
+					"regex": "^I[A-Z]",
+          "match": false
+        }
       }
     ],
-    "linebreak-style": ["error", "windows"],
-    "arrow-body-style": ["warn", "as-needed"],
-    "arrow-parens": ["error", "always"],
-    "no-unused-vars": [
-      "error",
-      {
-        "args": "none"
-      }
-    ],
-    "radix": ["error", "as-needed"],
-    "prefer-destructuring": "off"
-  },
-  "plugins": ["import", "prettier"],
-  "settings": {
-    "import/ignore": ["node_modules", ".png$", ".jpg$"]
+    "@typescript-eslint/explicit-module-boundary-types": "off",
+    "@typescript-eslint/member-ordering": "error",
+    "@typescript-eslint/no-explicit-any": "error",
+		"@typescript-eslint/no-use-before-define": ["error", { "functions": false, "variables": false }],
+    "@typescript-eslint/typedef": "error",
+    "@typescript-eslint/type-annotation-spacing": "error",
+    "@typescript-eslint/semi": "error",
+    "@typescript-eslint/no-var-requires": "off",
+    "no-useless-escape": "off",
+    "no-restricted-globals": "off",
+    "no-restricted-properties": "off",
+    "spaced-comment": "error",
+    "curly": ["error", "multi-line"],
+    "eol-last": ["error", "always"],
+    "guard-for-in": "error",
+    "valid-jsdoc": ["error", { "requireParamDescription": false, "requireReturn": false, "requireReturnType": false }],
+    "no-unused-labels": "error",
+    "max-len": "off",
+    "no-caller": "error",
+    "no-bitwise": "error",
+    "no-console": ["error", { "allow": ["log", "warn", "error", "info"] }],
+    "no-multiple-empty-lines": "error",
+    "no-new-wrappers": "error",
+    "no-debugger": "error",
+    "no-empty": "error",
+    "no-eval": "error",
+    "dot-notation": "error",
+    "no-fallthrough": "error",
+    "no-trailing-spaces": "error",
+    "no-unused-expressions": ["error", { "allowShortCircuit": true, "allowTernary": true }],
+    "brace-style": "error",
+    "quotes": ["error", "single"],
+    "radix": "error",
+    "default-case": "error",
+    "eqeqeq": "error"
```

Comment:
Add [jsx-quotes](https://eslint.org/docs/rules/jsx-quotes)? Either here or in react projects (preferred - along with moving the `ecmaFeatures: { jsx: true }` defined above). Probably prefer double. i.e.
```
'jsx-quotes': ['error', 'prefer-double'],
```

Reply:
Added